### ISSUE TITLE
Progress display issue

### ISSUE THREAD

**Issue by Zeek05** [Batch-Man/Progress#2]
The progress bar will create a mess if it is more than 3 less the maximum bar width.
The progress function should be using the maximum width of the screen (82 cells in the example below), however it is not.
On the call of the function, there is also a space before the `[` in the bar.

``Mode 82,25
For /l %%A in (0,1,1000) do (Call Progress 82 %%A 1000)
``

Will generate:
![image](https://user-images.githubusercontent.com/61340927/152459649-6361bd07-d8ac-4bad-a56a-5c38ed10a25a.png)
Notice the space on the far left as well.

The current workaround is to have your progress bar length be **at least N-3** where N is the length of your CMD.

**Comment by TheKvc** [comment]
Yes, this is a known issue. I wanted to integrate the auto checking of the screen size - but sometimes MODE command delays the result. So, I kept this up to the user to manage the numbers.

but, your concern makes sense to me. Is there any plugin (apart from MODE) which can spit out console size without much delay?
